### PR TITLE
Feature/transfer slot pickers to open source

### PR DESCRIPTION
**Pull request by Leooo**
- [x] change folder structure to have namespace `slots-picker`, `slots-filter`, `date-picker`, `clock-reloader`, `horizontal-list-swiper` packs.

Status after this PR:

- [x] transfer slot-picker components to separate, open-source repo. 
- [x] move sly and pickadate bower dependencies out of commons and into npm packages in this repo.
- [x] ~find a solution for `bg-button`, services, helpers and 'scroll-anchor` (should probably not be part of this repo)~ need to be able to tree-shake it out (make a special bg tree-shaking option)
- [ ] allow to tree-shake components
- [ ] allow to tree-shake dependencies
- [ ] create an ember-commons + one app PRs consuming it for backward compatibility
- [ ] make less dependency optional
- [x] reorganize folder structures to have everything under a single namespace instead of flattened components hierarchy
- [x] ~rename the components to `appointment-slots-picker`~
- [ ] make sure the latest LTS versions are used in ember-try
- [x] check font-awesome
- [x] github pages
- [ ] check slot-picker/loader image

### CODE REVIEW

**Comment by omersaleem** [comment]
@Leooo feature to mater?

**Comment by Leooo** [comment]
Yeah no mistake thanks

**Review by omersaleem — APPROVED** [review]

File: addon/styles/global-rules.less [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    //-moz-border-radius: 40px;
+    border-radius: 40px;
+    padding: 7px 30px;
+    border: 0;
+    box-shadow: none;
+    display: inline-block;
+  }
+
+  .btn-secondary {
+    color: #005eb8;
```
Comment (NikkiAlgarCentrica) [inline addon/styles/global-rules.less]: is there a variable for this colour?

File: addon/styles/global-rules.less [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    background: 0 0;
+    border: 2px solid #005eb8;
```
Comment (NikkiAlgarCentrica) [inline addon/styles/global-rules.less]: this one too...

File: addon/styles/global-rules.less [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+    padding: 5px 28px;
+    text-decoration: none;
+  }
+
+  .btn-primary {
+    color: #333f48;
+    background: #b3e000;
```
Comment (NikkiAlgarCentrica) [inline addon/styles/global-rules.less]: Are there variables for these hardcoded colours?

Comment (Leooo) [reply addon/styles/global-rules.less]: @NikkiAlgarCentrica this is a PR for an open-source addon, so can't use bg-vi - so will need to tree-shake out this CSS when importing into  connectedHomes repo (or put the CSS in the dummy app).

I guess those should be added to https://github.com/britishgas-engineering/ember-appointment-slots-pickers/blob/feature/transfer-slot-pickers-to-open-source/addon/styles/variables.less though, I will add it as a bullet point if that's fine.

Comment (Leooo) [reply addon/styles/global-rules.less]: @NikkiAlgarCentrica fixed thanks

**Review by NikkiAlgarCentrica — APPROVED** [review]

**Pull request merged**